llvm/lib/TargetParser/RISCVISAInfo.cpp

```diff
@@ -1112,67 +856,8 @@ struct ImpliedExtsEntry {
   bool operator<(StringRef Other) const { return Name < Other; }
 };
 
-// Note: The table needs to be sorted by name.
```

Comment:
IIRC there may have been some things that depended on the order here, hence the comment. But the PR description says the order is different. Does that break anything?

Reply:
The patch changed the order of the strings in the `ImpliedExts*` arrays above. But the order in those arrays don't matter.